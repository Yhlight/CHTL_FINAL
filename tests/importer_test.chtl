// tests/importer_test.chtl

// Use a direct relative path from the project root
[Import] @Chtl from "tests/module/definitions.chtl";

html {
    head {
        title { text: "Importer Test"; }
    }
    body {
        div {
            style {
                @Style CommonText;
                border: 1px solid black;
            }
            text: "This text should be styled by the imported module.";
        }
    }
}
