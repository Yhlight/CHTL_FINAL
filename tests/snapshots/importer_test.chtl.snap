<html>
  <head>
    <title>
      Importer Test
    </title>
  </head>
  <body>
    <div style="border: 1px; color: #333; font-family: Arial; font-size: 16px; ">
      This text should be styled by the imported module.
    </div>
  </body>
</html>
